.github/workflows/integration-tests.yml

```diff
@@ -117,3 +117,23 @@ jobs:
       #    docker run --rm -e COMPOSER_ROOT_VERSION -v $(pwd):/app -v $(which composer):/usr/local/bin/composer -v $(which vulcain):/usr/local/bin/vulcain -w /app php:8.0-alpine ./phpunit src/Symfony/Component/HttpClient/Tests/CurlHttpClientTest.php --filter testHttp2Push
       #    sudo rm -rf .phpunit
       #    [ -d .phpunit.bak ] && mv .phpunit.bak .phpunit
+
+  translation-status:
```

Comment:
I would prefer moving this into a new GitHub workflow and only triggering it when a translation file is changed:

```yaml
on:
  pull_request:
    paths:
      - 'src/**/Resources/translations/*.xlf'
```

Reply:
On my side, I think I'd prefer having it on an existing job as a new step.
The reason is that the command takes 0s, while checking out/etc takes a few more seconds.

Reply:
Both options are fine by me, but seem conflicting. 

What @nicolas-grekas proposes would not require making the CI configuration aware of the files that are checked by the script.

Feel free to let me know what approach is preferred.

Reply:
Both serve a different goal: don't let PRs fail that didn't do anything with translations (like we also do with coding standards) or improve reporting speed and reduce some CPU usage.

I think improving false failures is the way to go, especially if we want to introduce more policy checks like you suggested in https://github.com/symfony/symfony/issues/39055#issuecomment-884155608 . But let's see what the others think :)

Reply:
In addition, imho CI not being green is a temporary matter of fixing all mismatches asap after this PR gets merged.

Reply:
Can't we wrapping that command in a regular test case? That'd be the simplest to run this, even for future contributors.

Reply:
One of future's enhancement could be to make use of the `GithubActionReporter` to annotate the PR's problems in the translation files directy. I'm not sure how to achieve that when running the command wrapped in a PHPUnit test case.

Reply:
I'd like to start working on a PR that fixes all these translation id mismatches.

@nicolas-grekas @wouterj would it be possible to decide on what approach to follow, so we can get this PR merged?

Reply:
Maybe we can try the following action to keep it as a simple step instead of a separate workflow:

https://github.com/MarceloPrado/has-changed-path

Reply:
Implemented the suggestion of @OskarStark in fb731ac7c6a8b8cba97a4d95747ab0578e1bfa23 Running the script thereby became part of the existing integration tests.